zetaclient/chains/ton/liteapi/client.go

```diff
@@ -203,6 +224,10 @@ func (c *Client) getLastTransactionHash(ctx context.Context, acc ton.AccountID)
 	return state.LastTransLt, state.LastTransHash, nil
 }
 
+func TransactionToHashString(tx *ton.Transaction) string {
```

Comment:
Can we maybe add comment? How it is different from `tx.Hash()`?
Do we need a pointer for the parameter?

Reply:
https://github.com/zeta-chain/node/pull/3020/commits/9487eb1aa34369df44c15e3603b68009eb6110a5